drivers/include/sx127x.h

```diff
@@ -192,21 +192,28 @@ enum {
 };
 
 /**
+ * @name    SX127X device descriptor boolean masks
+ * @{
+ */
+#define SX127X_LOW_DATARATE_OPTIMIZE_FLAG_MASK       (1 << 0)
+#define SX127X_ENABLE_FIXED_HEADER_LENGTH_FLAG_MASK  (1 << 1)
+#define SX127X_ENABLE_CRC_FLAG_MASK                  (1 << 2)
+#define SX127X_CHANNEL_HOPPING_FLAG_MASK             (1 << 3)
+#define SX127X_IQ_INVERTED_FLAG_MASK                 (1 << 4)
+#define SX127X_RX_CONTINUOUS_FLAG_MASK               (1 << 5)
```

Comment:
These are not masks, they are flags. So don't call them that. Also: can you try if setting the flags explicitly (i.e. `0x01`, `0x02`, `0x04`, etc) makes a different in the code size?